trailblazer/server/api.py

```diff
@@ -200,10 +204,10 @@ def post_get_latest_analysis():
 
 @blueprint.route("/find-analysis", methods=["POST"])
 def post_find_analysis():
-    """Find analysis using case_id, date, and status."""
+    """Find analysis using case id, date, and status."""
     post_request: Response.json = request.json
     analysis: Analysis = store.get_analysis(
-        case_id=post_request.get("case_id"),
+        case=post_request.get("case_id"),
```

Comment:
```suggestion
        case=post_request.get("case_id"),
```
it is the pet name here, I am not sure how easy it is to change since it is an api request. but this would be the `case_internal_id`.

Reply:
Using case_name as there is no internal case id in Traiblazer so it would not make sense and couple to cg too tigthly